starwars/src/App.js

```diff
@@ -1,16 +1,19 @@
-import React, { Component } from 'react';
-import './App.css';
+import React, { Component } from "react";
+import "./App.css";
+import "./components/StarWars.css";
```

Comment:
You're not using the styles in StarWars.css in App.js, so you can remove the import from this file.